api/src/main.rs

```diff
@@ -0,0 +1,17 @@
+mod handlers;
+
+use actix_web::{web, App, HttpServer};
+
+#[actix_rt::main]
+async fn main() -> std::io::Result<()> {
+    std::env::set_var("RUST_LOG", "actix_web=debug");
```

Comment:
What logging should we expect to see? I'm not seeing things in a CodeSpaces buddy build.

Reply:
I thought we would get logging in the Output for all HTTP calls, but I am not seeing it either. Will remove for now and we can handle logging all up as separate issues.